packages/frontend/src/components/Layout/NavigationSidebar.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from 'react'
+import { Link, useMatch } from 'react-router-dom'
+import { Text } from '@chakra-ui/react'
+import { SidebarContainer, SidebarItem } from '@opengovsg/design-system-react'
+import { LayoutNavigationContext } from 'contexts/LayoutNavigation'
+
+export default function NavigationSidebar() {
+  const { links, closeDrawer } = useContext(LayoutNavigationContext)
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
🤔 

... why? Don't suppress lints unless you have a really good reason - this is likely to cause weird bugs down the line even if it seems to work now

Reply:
ok this was code written previously and I believe that the lint is required cus `useMatch` is a hook and we are passing it into a function, this function itself has to start with a Capital letter as a react component or start with `use` as a react hook